File: src/my_py_pkg/my_pkg/oops_1.py
#To send message continuously :
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node

class MyNode(Node):
    def __init__(self):
        super().__init__("Py_test")
        self.counter_=0
        self.get_logger().info("Hello World ")
        self.create_timer(1.0,self.callback_timer)

    def callback_timer(self):
        self.get_logger().info("Hello World "+str(self.counter_))
        self.counter_+=1



def main(args=None):
    rclpy.init(args=args)
    node=MyNode()
    rclpy.spin(node)
    rclpy.shutdown()

File: src/my_cpp_pkg/src/oops_1.cpp
//To print a message using OOPS :

#include "rclcpp/rclcpp.hpp"
class MyNode : public rclcpp::Node{
    public : 
    MyNode(): Node("oops_test")
    {
        RCLCPP_INFO(this->get_logger(),"Hello World !!!");
    }
    private:
};

int main(int argc,char** argv){
    rclcpp::init(argc,argv);
    auto node = std::make_shared<MyNode>();
    rclcpp::spin(node);
    rclcpp::shutdown();
    return 0;
}